### DISCUSSION TITLE
[QUESTION] How can I add a skiasharp view to the MainPage.xaml?

### DISCUSSION THREAD

**Discussion by proficad** [mono/SkiaSharp#2186] [Q&A]
How can I add a skiasharp view to the MainPage.xaml?
I tried to add 

    <skia:SKCanvasView x:Name="skiaView" PaintSurface="OnPaintSurface"
                       EnableTouchEvents="True" Touch="OnTouch"
                       IgnorePixelScaling="True" />

but I do not know how to add the namespace.

Thanks
Vaclav

**Comment by mattleibow** [comment]
Just add this to the top of the file:

Xamarin.Forms:
```
xmlns:skia="clr-namespace:SkiaSharp.Views.Forms;assembly=SkiaSharp.Views.Forms"
```

Maui:
```
xmlns:skia="clr-namespace:SkiaSharp.Views.Maui.Controls;assembly=SkiaSharp.Views.Maui.Controls
```

**Comment by proficad** [comment]
Thank you, but now I get plenty of errors like 

```
>App.xaml : error :  : XamlC error XFC0000 : Cannot resolve type "Application".
1>App.xaml : error :  : XamlC error XFC0000 : Cannot resolve type "Application".
1>App.xaml : error :  : XamlC error XFC0000 : Cannot resolve type "Application".
1>AppShell.xaml : error :  : XamlC error XFC0000 : Cannot resolve type "Shell".
```

Could you please have a look?

https://github.com/proficad/Viewer_Maui.git

Thanks
Vaclav

**Comment by proficad** [comment]
Problem solved, I am not sure how, but now the project is working.
Thanks
Vaclav
